session_rpcserver.go

```diff
@@ -135,7 +135,6 @@ func (s *sessionRpcServer) start() error {
 				continue
 			}
 
-			ctx := context.Background()
 			ctxc, cancel := context.WithTimeout(
```

Comment:
it looks like not all `s.cfg.autopilot` queries have a timeout context, should we make this consistent?

Reply:
i thiiiink it is currently only the ones that are not directly called from an RPC method that dont do the timeout cause there is nothing else to cancel them. For rpc calls, if the rpc context is cancelled, the call is cancelled